### PR TITLE
Closes #45. Adds name filter to map view.

### PR DESCRIPTION
**Pull request by gotoplanb**
Animated GIF happypath steps:

1. Type number info field
2. Tap save button
3. See label is updated with value from field

gda-45

### CODE REVIEW

**Comment by gotoplanb** [comment]
Let me know if you'd like to hop on a screenshare and review.

**Comment by gotoplanb** [comment]
:shipit:

File: README.md [hunk 1]
```diff
@@ -1,20 +1,9 @@
...
-
-Another list:
-
-1. One
-2. Second
-3. And a third
+## Build
 
+1. Checkout the code
+2. `mnv test`
```
Comment (delkant) [inline README.md]: @gotoplanb what is this??

Comment (gotoplanb) [inline README.md]: @delkant I can see it.

**Comment by gotoplanb** [comment]
:shipit:

**Pull request merged**